content/docs/en/elements/components/side-drawer.md

```diff
@@ -0,0 +1,54 @@
+---
+title: SideDrawer
+apiRef: https://docs.telerik.com/devtools/nativescript-ui/Controls/NativeScript/SideDrawer/overview
+contributors: [MisterBrownRSA, rigor789, eddyverbruggen, ikoevska, sombriks]
+---
+
+`<RadSideDrawer>` is a UI side drawer component which plays a navigation role on application structure.
+
+## Installing the side drawer:
+
```

Comment:
`<RadSideDrawer>` is not available with the core NativeScript installation. You need to add it as a plugin to your project.